app/src/main/java/waiter/Connectable/ClientConnection.java

```diff
@@ -20,11 +21,16 @@ public ClientConnection(Socket socket) throws IOException {
     }
 
     public String read() throws IOException {
-        return this.bufferedReader.readLine();
+        StringBuilder requestContents = new StringBuilder();
+        while(!requestContents.toString().contains("\r\n")) {
```

Comment:
So this will stop reading input once a CRLF is encountered? I think you might bump into issues with this in the future if I'm understanding this correctly, but if it is working with the current server specs I'm fine with approving this as is.

Reply:
It reads until it hits the first CRLF, so it reads the `GET  /simple_get HTTP/1.1` and then encounters a "\r\n" so it stops reading and then strips the line to work properly currently. I'll keep it like this for now like you said since it is indeed working, but I will also keep an eye out for future problems I might encounter. Thank you for both the help and the heads up 😄